### PR TITLE
fix(webapp): split privateKey for ghost-admin

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Split `privateKey` for ghost-admin to `id` and `secret`

### CODE REVIEW

File: packages/webapp/src/pages/Connection/Create.tsx [hunk 1]
```diff
@@ -463,21 +463,28 @@ export default function IntegrationCreate() {
...
             if (credentials.length > 0) {
                 jwtCredentialsString = `
-                credentials: {
-                    ${credentials.join(',\n            ')}
-                }
-                `;
+    credentials: {
+        ${credentials.join(',\n            ')}
+    }
+    `;
```
Comment (TBonnin) [inline packages/webapp/src/pages/Connection/Create.tsx:501]: formatting is wrong here right?

**Review by TBonnin — APPROVED** [review]

File: packages/webapp/src/pages/Connection/Create.tsx (hunk 1 above)
Comment (hassan254-prog) [reply packages/webapp/src/pages/Connection/Create.tsx:501]: yes, formatting was initially wrong, with `ghost-admin`  we need to split the `apiKey` to `id` and `secret` and at the webapp

Comment (TBonnin) [reply packages/webapp/src/pages/Connection/Create.tsx:501]: no I mean the tabulations change of the 4 lines above has been committed by accident

Comment (bodinsamuel) [reply packages/webapp/src/pages/Connection/Create.tsx:501]: it's a template string for display

Comment (hassan254-prog) [reply packages/webapp/src/pages/Connection/Create.tsx:501]: yes, for indentation.

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**